js/globals.ts

```diff
@@ -1,5 +1,25 @@
 // Copyright 2018 the Deno authors. All rights reserved. MIT license.
 
+import { Console } from "./console";
+
+declare global {
+  type MessageCallback = (msg: Uint8Array) => void;
+
+  interface Deno {
+    print(text: string): void;
+    recv(cb: MessageCallback): void;
+    send(msg: ArrayBufferView): Uint8Array | null;
+  }
+
+  interface Window {
```

Comment:
I wonder -- is 'window' currently not the same as the global object?

Reply:
Hmm ... not sure. how is that should be expressed in declaration files? 
Anyway - it’s orthogonal to this PR. Landing.